communities/src/benchmarking.rs

```diff
@@ -156,8 +158,8 @@ benchmarks! {
 		// Todo: Properly benchmark this #189
 
 		let (cid, bootstrappers, community_metadata, demurrage, nominal_income) = setup_test_community::<T>();
-		let mut cids = vec![CommunityIdentifier::default(); 1_000_000];
-		cids.push(cid);
+		let mut cids: BoundedVec<CommunityIdentifier, T::MaxCommunityIdentifiers> = bounded_vec![CommunityIdentifier::default(); 9];
```

Comment:
because the limit in the bounded vec limit in testruntime is set to 10. we could increase it but then some other tests would get nasty if i remember correctly.

Reply:
Hmm, ok. I doubt that it will influence the weight greatly anyhow, so I am fine with this.